web_src/js/utils.ts

```diff
@@ -39,8 +39,9 @@ export function parseIssueHref(href: string): IssuePathInfo {
 
 export function parseIssueNewHref(href: string): IssuePathInfo {
   const path = (href || '').replace(/[#?].*$/, '');
-  const [_, ownerName, repoName, pathType, indexString] = /([^/]+)\/([^/]+)\/(issues|pulls)\/new/.exec(path) || [];
-  return {ownerName, repoName, pathType, indexString};
+  const [_, ownerName, repoName, pathTypeField] = /([^/]+)\/([^/]+)\/(issues\/new|compare\/.+\.\.\.)/.exec(path) || [];
```

Comment:
Why only `...`? Isn't `..` a valid comparison as well?
Also, what happened to `|pulls` and the `indexString`?

Reply:
* `...` is for merge base, it is used for creating PR
* there is no `|pulls`
* there is no `indexString`